run_model-dep-full_limits_per_era.sh

```diff
@@ -90,13 +94,14 @@ elif [[ $MODE == "ws" ]]; then
     ############
     # workspace creation
     ############
-    combineTool.py -M T2W -o ws_mh125.root \
+    combineTool.py -M T2W -o ${wsoutput} \
```

Comment:
I think this variable is not set in this script as the mh125 model is hard-coded.

Reply:
ok yes, should be changed

Reply:
done